webapp-django/crashstats/documentation/templates/documentation/supersearch/home.html

```diff
@@ -0,0 +1,389 @@
+                    </tr>
+                    <tr>
+                        <td>API endpoint</td>
+                        <td><a href="{{ url('api:model_wrapper', model_name='SuperSearch') }}">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</a></td>
+                    </tr>
+                    <tr>
+                        <td>Private endpoint</td>
+                        <td><a href="{{ url('api:model_wrapper', model_name='SuperSearchUnredacted') }}">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearchUnredacted') }}</a></td>
+                    </tr>
+                    <tr>
+                        <td>API Documentation</td>
+                        <td><a href="{{ url('documentation:supersearch_api') }}">{{ full_url(request, 'documentation:supersearch_api') }}</a></td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <h1 id="what-is-super-search">What is Super Search?</h1>
+
+            <p>Super Search is an interface to crash reports data. There is a <a href="{{ url('supersearch.search') }}">human-friendly interface</a> and a <a href="{{ url('api:documentation') }}#SuperSearch">public API</a>. This guide covers the API, but the interface can be used to easily query and visualize the data.</p>
+            <p>There are 3 key points to Super Search. First, it is a secured window on all the raw crash reports data we have. Second, it allows you to use almost every field as a filter, giving you great power to explore the huge data we have. And third, it has advanced aggregation features that can be used to extensively analyze a subset of data.</p>
+            <p>This documentation aims at extensively covering the public API.</p>
+
+            <h1 id="api-endpoints">API endpoints</h1>
+
+            <h2 id="supersearch">SuperSearch</h2>
+
+            <pre><code><span class="http-verb">GET</span> <span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span></code></pre>
+
+            <p>This is the public endpoint that you should mostly be using. It doesn't require any permissions, and gives you access to almost all of the data about crash reports. The only restrictions are fields potentially containing sensitive data (like PII, for Personally Identifiable Information), such as <code>email</code> or <code>url</code>.</p>
+
+            <h2 id="supersearchunredacted">SuperSearchUnredacted</h2>
+
+            <pre><code><span class="http-verb">GET</span> <span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearchUnredacted') }}</span></code></pre>
+
+            <p>This is a restricted endpoint, only accessible through the use of an API Token and your having the right set of permissions. It gives you an exhaustive access to our entire dataset, so please use this very carefully.</p>
```

Comment:
The word "API Token" should perhaps link to `/api/tokens` (using `reverse()` of course)